docs/cloud/index.md

```diff
@@ -4,50 +4,28 @@ title: Cloud Services
+CSC Cloud Services consists of Pouta hardware virtualisation service, Rahti application containerisation service, Pukki cloud database service and CSC Notebooks web application course hosting service. CSC hosts all services itself, with data centres in Finland.
 
 ### Pouta
 
-With Pouta services, you can can take control of your IT infrastructure, manage it in a flexible fashion, and automate it as you require. You can launch and manage virtual machines, deploy storage, and create networks for your needs. This infrastructure can be used for anything you need or can imagine.[^2]
-
-<iframe
-    width="400"
-    height="300"
-    srcdoc="https://www.youtube.com/embed/2opuVIBSPgw"
-    title="CSC's Pouta Cloud Services"
-    frameborder="0"
-    allow="accelerometer; autoplay; clipboard-write; encrypted-media;gyroscope; picture-in-picture"
-    allowfullscreen>
-</iframe>
+The Pouta service is a cloud computing infrastructure based on the OpenStack Rocky release. It provides virtual computer hardware platforms, storage and computer network resources.
 
-More information about using Pouta may be found from [here](pouta/).
+If you need a full access to your virtual machines, storage and virtual network devices, choose [Pouta](pouta/).
 
-### Pukki - Database as a Service
+### Pukki
 
-Pukki is a service that provides on demand databases with automatic backups and maintenance. If you have a CSC-account you should be able to get your first database within 15 minutes and your second database within five. If you are doing research, arranging courses or building a new services that needs databases you probably want to check out this service before you start setting up manually your own database.
+Pukki is a cloud database service that provides on demand databases with automatic backups and maintenance. If you have a CSC-account you should be able to get your first database within 15 minutes and your second database within five. If you are doing research, arranging courses or building a new services that needs databases you probably want to check out this service before you start setting up manually your own database.
 
-More information about using Pukki may be found from [here](dbaas/).
+If you need a managed database server with backups, choose [Pukki](dbaas/).
 
 ### Rahti
 
-Rahti makes the deployment of applications a breeze. This is a higher level service than Pouta services, and Rahti automatically takes care of a lot of the lower level management, so that you don't have to.
+Rahti is a containerisation service based on Red Hat OKD version 3.11. It provides application level virtualisation where software applications can run in isolated user spaces without root access.
```

Comment:
At least for Pouta I think "real customers"  don't care that much about the version maybe this is different in OKD.

Reply:
Removed - for the better of real customers! 😁